pages/blog/app-refresh-grouparoo-and-dbt.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "Use App Refresh to run Grouparoo with dbt"
+date: "2021-12-09"
+author: "Teal Larson"
+pullQuote: "Learn more about using App Refresh Queries to trigger schedules and learn how to integrate this feature with a dbt project."
+seoDescription: "Looking to get your data from your dbt Projects to your Destinations? The Grouparoo experts will walk you through how to get started. ✓ Start a free trial now!"
+image: "app-refresh-grouparoo-and-dbt/211130-dbt-grouparoo.png"
+tags: [data, engineering]
+---
+
+As part of our goal to make Data Engineer's lives easier by integrating more deeply with the Modern Data Stack, we've been working on new ways to trigger Grouparoo's Schedules to run. One such use case we've been especially interested in is how to trigger a Grouparoo run based off the completion of a dbt run. This year we've watched dbt take off as the open source data transformation tool of choice and were excited for their 1.0 release this month(https://blog.getdbt.com/dbt-core-v1-is-here/)! Giving a Grouparoo App the power to check the data warehouse for a specific query and trigger its Schedules whenever new data is found opens up new opportunities for integrating with projects who are already performing robust ELT processes to be able to continue the cycle and export cohorts of data to operational tools via Reverse ETL.
```

Comment:
```suggestion
As part of our goal to make Data Engineer's lives easier by integrating more deeply with the Modern Data Stack, we've been working on new ways to trigger Grouparoo's Schedules to run. One such use case we've been especially interested in is how to trigger a Grouparoo run based off the completion of a dbt run. This year we've watched dbt take off as the open source data transformation tool of choice and were excited for their [1.0 release this month](https://blog.getdbt.com/dbt-core-v1-is-here/)! Giving a Grouparoo App the power to check the data warehouse for a specific query and trigger its Schedules whenever new data is found opens up new opportunities for integrating with projects that are already performing robust ELT processes. Doing so allows Grouparoo users to be able to continue the modern data stack flow and to export cohorts of data to operational tools via Reverse ETL.
```

I tried to split up this sentence because it's a bit long. I'm not super happy with the split, so I'm definitely open to further iteration!

Reply:
Agree!  I'm going to commit this and then see what else I can come up with.

Reply:
```Our new App Refresh Query gives Grouparoo Apps the power to query their data warehouse every minute for new data.  This opens up opportunities to integrate with existing ELT projects that use tools like dbt. Grouparoo users are now able to continue the modern data stack flow and to export cohorts of data to operational tools via Reverse ETL.```

Maybe?  Splits it into three sentences...